source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.integration.BoTorchSampler.po

```diff
@@ -128,25 +138,26 @@ msgid ""
 " return value must be a sequence of :obj:`float` s. A value strictly "
 "larger than 0 means that a constraint is violated. A value equal to or "
 "smaller than 0 is considered feasible."
-msgstr ""
+msgstr "可选的用于计算目标函数约束条件的函数. 其接受 :class:`~optuna.trial.FrozenTrial` 作为参数, 返回约束条件. "
```

Comment:
This is duplicated, the same as line 130

Reply:
Generated by Sphinx and just removed it.